scipy/optimize/tests/test_linprog.py

```diff
@@ -2423,6 +2476,20 @@ def test_semi_continuous(self):
         np.testing.assert_allclose(res.x, [0, 0, 1.5, 1])
         assert res.status == 0
 
+    def test_bug_20584(self):
+        """
+        Test that when integrality is a list of all zeros, linprog gives the
+        same result as when it is an array of all zeros / integrality=None
```

Comment:
Not sure what the SciPy policy is, but would be nice to have a link to the issue.